src/Hl7.Fhir.Base/Serialization/engine/FhirSerializationEngineFactory.cs

```diff
@@ -111,50 +114,67 @@ private static FhirJsonParsingSettings buildJsonParsingSettings(Mode mode) => ne
 
 
         /// <summary>
-        /// Create an implementation of <see cref="IFhirSerializationEngine"/> configured to flag all parsing errors, 
+        /// Create an implementation of <see cref="IFhirExtendedSerializationEngine"/> configured to flag all parsing errors, 
```

Comment:
I don't think I want to make this interface public. So maybe it is better for these to return IFhirSerializationEngine, and then in the FhirJsonConverter, we can assume/enforce that the engine implements `IFhirExtendedSerializationEngine` or that it is actually a `PocoSerializationEngine`.